crates/burn-train/src/metric/base.rs

```diff
@@ -63,7 +63,7 @@ pub trait Metric: Send + Sync {
 /// Adaptor are used to transform types so that they can be used by metrics.
 ///
 /// This should be implemented by a model's output type for all [metric inputs](Metric::Input) that are
-/// registered with the [leaner buidler](crate::learner::LearnerBuilder) .
+/// registered with the [leaner builder](crate::learner::LearnerBuilder) .
```

Comment:
Looks like there was another typo hidden in plain sight 😅 

```suggestion
/// registered with the [learner builder](crate::learner::LearnerBuilder) .
```